src/components/transactionsV2/filters/messageFieldGroup.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { translate } from 'react-i18next';
+import { InputV2 } from '../../toolbox/inputsV2';
+import styles from './filters.css';
+
+class MessageFieldGroup extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      fields: {
+        message: {
+          value: '',
+          error: false,
+        },
+      },
+      feedback: '',
+    };
+
+    this.handleFieldChange = this.handleFieldChange.bind(this);
+  }
+
+  handleFieldChange({ target }) {
+    const { t } = this.props;
+    const { fields } = this.state;
+    const messageMaxLength = 62;
+    const error = encodeURI(target.value).split(/%..|./).length - 1 > messageMaxLength;
```

Comment:
it is unclear for me what is this split doing. Could you please add comment or sth that will explain what is going on

Reply:
done ✅